client/pages/product/addproduct.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react';
+import { axiosInstance } from '../../configs/axios';
+import { Router } from 'next/router';
+
+const CreateProduct = () => {
+  const [product, setProduct] = useState({
+    name: '',
+    description: '',
+    image: '',
+    price: '',
+    interval: 'monthly', // Default interval
+  });
+
+  // Use state to store the token
+  const [storedToken, setStoredToken] = useState('');
+
+  useEffect(() => {
+    // Check if we are on the client side before accessing localStorage
+    const token = window.localStorage.getItem('token');
+    setStoredToken(token);
```

Comment:
This is wrong, token should not be exposed like this. 

Consider accessing the token while making an API call